packages/cli/src/commands/BaseCommand.ts

```diff
@@ -63,6 +63,10 @@ export abstract class BaseCommand extends Command {
 			this.exitWithCrash('There was an error initializing DB', error),
 		);
 
+		// This needs to happen after DB.init() or otherwise DB Connection is not
+		// available via the dependency Container that services depend on.
+		this.shutdownService.validate();
```

Comment:
Should we do this only during `development` or `test` to avoid adding additional startup delays in `production`?

Reply:
Done